test/go_modules/test_repo/pleasew

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
do you need this file? I assume we wouldn't actually use it for the test?

Reply:
Nope! Sorry, i used `plz init` to create this repo!